src/main/java/io/temporal/samples/hello/HelloPeriodic.java

```diff
@@ -88,14 +94,35 @@
   // Define the workflow implementation which implements the greetPeriodically workflow method.
   public static class GreetingWorkflowImpl implements GreetingWorkflow {
 
+    // In this example, we use an internal of 5 seconds with an intended variation of +/- 1 second
+    // between executions of some useful work. In real applications a higher value may be more
+    // appropriate, for example one that matches a business cycle of several hours or even days.
+    private static class ScheduleConfig {
```

Comment:
I would remove this class as it doesn't add any value as it is just a holder of constants. Why is SingleWorkflowRunIterations are not part of any class.

Reply:
✔️